docs/reference/builder.md

```diff
@@ -484,15 +484,14 @@ To view an image's labels, use the `docker inspect` command.
 
     EXPOSE <port> [<port>...]
 
-The `EXPOSE` instructions informs Docker that the container will listen on the
+The `EXPOSE` instructions informs Docker that the container listens on the
 specified network ports at runtime. Docker uses this information to interconnect
-containers using links (see the [Docker User
-Guide](../userguide/dockerlinks.md) and to determine which ports to expose to the
-host when [using the -P flag](run.md#expose-incoming-ports).
+containers using links (see [Linking containers together](../userguide/dockerlinks.md))
+and to set up port redirection on the host system when [using the -P flag](run.md#expose-incoming-ports).
 
 > **Note**:
-> `EXPOSE` doesn't define which ports can be exposed to the host or make ports
-> accessible from the host by default. To expose ports to the host, at runtime,
+> `EXPOSE` doesn't make the ports of the container accessible to host by
```

Comment:
Just had a quick chat with @moxiegirl and we think this doesn't have to be a "NOTE", because it's not really exceptional, but just what `EXPOSE` does (not) do.

@moxiegirl pointed me to another description for EXPOSE that she wrote, and that may better describe what it actually does; see the first paragraph of this section: https://docs.docker.com/reference/run/#expose-incoming-ports